File: components/permissions/src/main/java/org/cloudfoundry/credhub/requests/PermissionsV2PatchRequest.java
package org.cloudfoundry.credhub.requests;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonAutoDetect;
import org.cloudfoundry.credhub.PermissionOperation;

@JsonAutoDetect
public class PermissionsV2PatchRequest {
  private List<PermissionOperation> operations;

  public List<PermissionOperation> getOperations() {
    return operations;
  }

  public void setOperations(final List<PermissionOperation> operations) {
    this.operations = operations;
  }
}

File: components/permissions/src/main/kotlin/org/cloudfoundry/credhub/requests/PermissionsV2PatchRequest.kt
package org.cloudfoundry.credhub.requests

import com.fasterxml.jackson.annotation.JsonAutoDetect
import org.cloudfoundry.credhub.PermissionOperation

@JsonAutoDetect
class PermissionsV2PatchRequest {
    var operations: MutableList<PermissionOperation>? = null
}
